src/SymfonyRequirements.php

```diff
@@ -20,9 +20,10 @@
  */
 class SymfonyRequirements extends RequirementCollection
 {
-    const REQUIRED_PHP_VERSION = '5.5.9';
+    const REQUIRED_PHP_VERSION_BELLOW_SF4 = '5.5.9';
```

Comment:
typo: BELOW

Reply:
What about `REQUIRED_PHP_VERSION_3x` and `REQUIRED_PHP_VERSION_4x` (I know one can use this lib on 2.x, but as this is an internal detail, I think we're good.)

Reply:
Good suggest for `REQUIRED_PHP_VERSION_3x` and `REQUIRED_PHP_VERSION_4x`, I renamed the two constants.